config/routes.rb

```diff
@@ -1,8 +1,12 @@
 Rails.application.routes.draw do
-  root to: "users#index"
+  root to: "dashboard#index"
 
   devise_for :users
   resources :users, except: [:destroy, :show]
 
-  resources :books, except: [:show]
+  resources :books, except: [:show] do
+    member do
+      patch :toggle_featured
```

Comment:
`patch` - HTTP verb

Use `PUT`/`PATCH` when you are updating a resource

Use `POST` when you are creating a resource

Use `DELETE` when you are deleting a resource

Use `GET` when you are retrieving resource (EG: index/show pages)